### PR TITLE
Add shortcut for messaing semconv messaging.operation.send

### PR DESCRIPTION
**Pull request by hstan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - OTLP Metric exporter supports Histogram aggregation. (#1209)
+- Added messaging operation semconv shortcut for `messaging.operation.send`
```
Comment (XSAM) [inline CHANGELOG.md:14]:
```suggestion
- Added messaging operation semconv shortcut for `messaging.operation.send`. (#1215)
```

**Review by XSAM — APPROVED** [review]
Looks good. Thanks for your contribution.

**Comment by hstan** [comment]
Thanks @XSAM I've fixed the CLA. 
However, the build is failing due to a seemingly flaky test, I don't have permission to rerun it.

**Review by MrAlias — CHANGES_REQUESTED** [review]
[The specification for messaging semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/messaging.md#messaging-attributes) appears to explicitly state this value should not be set:

> If the operation is "send", this attribute MUST NOT be set, since the operation can be inferred from the span kind in that case.

Given that is the case we probably do not want to include this value here.

**Review by Aneurysm9 — APPROVED** [review]

**Comment by MrAlias** [comment]
Closing due to the incompatibility with the specification.

**Pull request closed without merging**